run_snafu.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
In most of our other workloads we symlink python3 to python... Any specific reason we are changing that here?

Reply:
To avoid that link the future :) and be able to execute snafu as ./run_snafu.py